src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -678,6 +678,9 @@ void GeneralPage::initLayout(SettingsLayout &layout)
     layout.addCheckbox(
         "Hide viewercount and stream length while hovering the split",
         s.hideViewerCountAndDuration);
+    layout.addCheckbox(
```

Comment:
Instead add a new section similar to the following:

> ### Streamer Mode
> 
> Chatterino can automatically change behavior if it detects that "OBS Studio" running. Select which things you? want to change while streaming:
> 
> [ ] Hide unsafe user content (Profile pictures and thumbnails)

This allows people to understand what the feature does and how it triggers (obs being open).

Reply:
Resolved in b9503b7